eureka-core-jersey3/src/main/java/com/netflix/eureka/transport/Jersey3RemoteRegionClientFactory.java

```diff
@@ -0,0 +1,117 @@
+package com.netflix.eureka.transport;
+
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+
+import com.netflix.discovery.shared.resolver.EurekaEndpoint;
+import com.netflix.discovery.shared.transport.EurekaHttpClient;
+import com.netflix.discovery.shared.transport.TransportClientFactory;
+import com.netflix.discovery.shared.transport.jersey3.EurekaIdentityHeaderFilter;
+import com.netflix.discovery.shared.transport.jersey3.EurekaJersey3Client;
+import com.netflix.discovery.shared.transport.jersey3.EurekaJersey3ClientImpl;
+import com.netflix.discovery.shared.transport.jersey3.Jersey3ApplicationClient;
+import com.netflix.eureka.EurekaServerConfig;
+import com.netflix.eureka.EurekaServerIdentity;
+import com.netflix.eureka.GzipEncodingEnforcingFilter;
+import com.netflix.eureka.resources.ServerCodecs;
+import jakarta.inject.Inject;
+import jakarta.ws.rs.client.Client;
+import jakarta.ws.rs.core.Feature;
+import jakarta.ws.rs.core.MultivaluedHashMap;
+import org.glassfish.jersey.message.GZipEncoder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class Jersey3RemoteRegionClientFactory implements TransportClientFactory {
```

Comment:
This implementation came from the now deleted `EurekaServerHttpClients` https://github.com/Netflix/eureka/pull/1473/files#diff-bcd6df8c6a0b1c6ebf2d76ccb55219b364009609e9b57df874f9401f31faa2be